en/user_guide/user.rst

```diff
@@ -0,0 +1,50 @@
+User, Group and Role
+====================
+
+Open Orchestra offer the possibility to manage different User with different right.
+
+User Management
+---------------
+
+Page Users in Administration allow to search,create,edit ans delete site users.
+
+At the creation, some fields are required :
+    - First Name
+    - Last Name
+    - E-mail
+    - Username
+    - Password
+
+After it's possible to Edit User to choose this default language which is display on connect and affiliate himself to some group.
```

Comment:
Once the user is created, you can add some information regarding his default language, his group, ...